docker-images/backend/Dockerfile

```
@@ -0,0 +1,46 @@
+FROM python:3.11.10
+
+ENV DEDUG=False
+
+# Prevents Python from writing pyc files.
+ENV PYTHONDONTWRITEBYTECODE=1
+
+# Keeps Python from buffering stdout and stderr to avoid situations where
+# the application crashes without emitting any logs due to buffering.
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR /app
+
+# Create a non-privileged user that the app will run under.
+# See https://docs.docker.com/go/dockerfile-user-best-practices/
+ARG UID=10001
+RUN adduser \
+    --disabled-password \
+    --gecos "" \
+    --home "/nonexistent" \
+    --shell "/sbin/nologin" \
+    --no-create-home \
+    --uid "${UID}" \
+    appuser
+
+# Copy the source code into the container.
+COPY manage.py Makefile pyproject.toml poetry.lock /app/
+COPY task_manager /app/task_manager
+COPY locale /app/locale
+
+# Install Poetry
+RUN pip install poetry
+
+# Skipping virtualenv creation
+RUN poetry config virtualenvs.create false
+
+# Install dependencies
+RUN poetry install
+
+# Switch to the non-privileged user to run the application.
+USER appuser
+
+# Expose the port that the application listens on.
+EXPOSE 8000
+
+CMD ["bash", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"]
```

app wasn't working until I made this change

```suggestion
CMD ["sh", "-c", "python manage.py migrate && python manage.py runserver 0.0.0.0:8000"]
```